src/__mocks__/WeatherMocks.tsx

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export const fetchWeatherData = async () => {
```

Comment:
This function does not really serve any purpose as you are mocking the API data in the tests themselves